contrib/http_servlet/src/main/java/io/opencensus/contrib/http/servlet/TraceConfigOptions.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.http.servlet;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import io.opencensus.common.ExperimentalApi;
+import io.opencensus.trace.Sampler;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.propagation.TextFormat;
+import io.opencensus.trace.samplers.Samplers;
+
+/**
+ * This class provide configuring options for Tracing.
+ *
+ * @since 0.18
+ */
+@ExperimentalApi
+public class TraceConfigOptions {
+  private TextFormat propagator = Tracing.getPropagationComponent().getB3Format();
```

Comment:
we should probably make this default to trace context (you can blame me on being lazy to finish that). Maybe add a TODO. We should not ship it with b3 because we will have to deal with deprecation and the whole process we do it in go

Reply:
I am removing this file (see above) but I'll add the TODO in other relevant place(s).